source/Government.cpp

```diff
@@ -147,6 +148,14 @@ void Government::Load(const DataNode &node)
 			enforcementZones.emplace_back(child);
 		else if(key == "provoked on scan")
 			provokedOnScan = true;
+		else if(child.Token(0) == "show title")
+			titleVisibility = TitleVisibility::SHOWN;
+		else if(child.Token(0) == "hide title")
+			titleVisibility = TitleVisibility::HIDDEN;
+		else if(child.Token(0) == "show title on log")
+			titleVisibility = TitleVisibility::SHOWN_ON_LOG;
+		else if(child.Token(0) == "reputation titles")
```

Comment:
This means we can have different ones depending on the government, right? However you dont see to define it in the datas, and show reputation without one defined should be showed as an error later on, I think.
Except if it somehow defaults to the one you provided but I dont think so (could be handy to have a default though)

Reply:
It should default to the global set if a specific set isn't provided.

Reply:
All good then I just didnt really see how since its instanticated empty in gov